src/test/kotlin/org/elm/ide/intentions/ElmMakeDeclarationIntentionTest.kt

```diff
@@ -34,21 +35,25 @@ f : (Int -> Int) -> List a -> (Char, String) -> { foo : Int } -> Bool{-caret-}
 """
                 , """
 f : (Int -> Int) -> List a -> (Char, String) -> { foo : Int } -> Bool
-f function list tuple record = {-caret-}
+f function list (char, string) record = {-caret-}
 """)
     }
 
 
     fun `test function parameters should be camelCased`() {
         doAvailableTest(
                 """
+type FooBar = FooBar
+type QuuxQuuxQuux = QuuxQuuxQuux
```

Comment:
The downside to using the `Ty` to generate params is that we don't have any information if the type isn't resolved. This also pops up occasionally in documentation rendering. Maybe I could add a new property to `TyUnknown` that contains the name of unresolved references so that we can show them anyway.